src/components/shared/Promotional/Promotional.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+export const Promotional = ({
+  title,
+  description,
+  features,
+  strongText,
+  actionText,
+  actionUrl,
+  iconUrl,
+  imageUrl,
+}) => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+  return (
+    <section className="dp-gray-page p-t-54 p-b-54">
+      <div className="dp-container">
+        <div className="dp-rowflex">
+          <div className="col-sm-12">
+            <div className="dp-icon-promotion">
+              <img src={_('common.ui_library_image', { imageUrl: iconUrl })} alt="icon" />
+            </div>
+          </div>
+          <div className="col-lg-6 col-md-12">
+            <div className="dp-content-promotion">
+              <h1>{title}</h1>
+              <p>{description}</p>
+
+              {features ? (
+                <ul className="dp-feature-list m-t-24">
+                  {features.map((feature, index) => (
+                    <li key={index} className="m-t-12">
+                      <span className="dp-icodot">.</span>
+                      <span>{feature}</span>
+                    </li>
+                  ))}
+                </ul>
+              ) : null}
+
+              <span className="dp-cta-paragraph">{strongText}</span>
```

Comment:
instead `strongText`, maybe `paragraph` sounds better. What do you think?

Reply:
Sounds good!